from discord.ext.commands import errors


class SendableException(errors.DiscordException):
    pass
